### DISCUSSION TITLE
When I execute make clean all

### DISCUSSION THREAD

**Discussion by Sakuya-CN** [filecoin-project/lotus#6741] [General Q&A]
When I execute make clean all, I always report an error. All the dependent packages have been installed. I really don't understand why this is the case? It cannot execute this command correctly
![image](https://user-images.githubusercontent.com/56291932/125416745-7bdc53c8-5a30-41ea-ace4-53b2cd4c6a11.png)

root@ipfs:/mnt/Paarams/lotus# make clean all
rm -rf  build/.filecoin-install build/.update-modules  lotus lotus-miner lotus-worker lotus-shed lotus-gateway lotus-seed lotus-pond lotus-townhall lotus-fountain lotus-chainwatch lotus-bench lotus-stats lotus-pcr lotus-health lotus-wallet lotus-keygen testground tvx lotus-sim
make -C extern/filecoin-ffi/ clean
make[1]: Entering directory '/mnt/Paarams/lotus/extern/filecoin-ffi'
make[1]: *** No rule to make target 'clean'.  Stop.
make[1]: Leaving directory '/mnt/Paarams/lotus/extern/filecoin-ffi'
make: [Makefile:310: clean] Error 2 (ignored)
git submodule update --init --recursive
touch build/.update-modules
make -C extern/filecoin-ffi/ .install-filcrypto
make[1]: Entering directory '/mnt/Paarams/lotus/extern/filecoin-ffi'
make[1]: *** No rule to make target '.install-filcrypto'.  Stop.
make[1]: Leaving directory '/mnt/Paarams/lotus/extern/filecoin-ffi'
make: *** [Makefile:37: build/.filecoin-install] Error 2

**Comment by Sakuya-CN** [comment]
The rest of the machines can successfully execute this command in the test, but some of them are not. I haven't found the problem so far. I hope you can give me a direction

**Comment by de-authority** [comment]
have you solve this?

**Comment by SgtCoin** [comment]
@Motto-chen Did you resolve this? If not, please let me know. I will be happy to help.
